[BUILD-624] chore: Adjust js message handlers, don't apply modifications to flashcard selector

RisingOrange opened the pull request:
This task makes some adjustments to the js message handlers to make the add-on work with this related web app PR: https://github.com/AnkiHubSoftware/ankihub/pull/2313

## Related issues
https://ankihub.atlassian.net/browse/BUILD-624


## Proposed changes
- Remove code that applies modifications to the web apps flashcard selector web page. After this web app PR this will not longer be necessary: https://github.com/AnkiHubSoftware/ankihub/pull/2313. With this web app RP the web app will call the `ankihub_sync_notes_actions` pycmd itself.
- Change format for arguments to the `ankihub_sync_notes_actions` pycmd
  - Previously it expected this format: `ankihub_sync_notes_actions <deck_id>`
  - Now the format is analogous to the other pycmds `ankihub_sync_notes_actions {"deckId": <deck_id>}`
- [perf: Use anki note ids in query for opening notes in browser](https://github.com/AnkiHubSoftware/ankihub_addon/pull/1010/commits/1d3b8532ec96978436d15e8a81348c0f01f993ac)
  - The other method was very slow for many note ids


## How to reproduce
See the how-to-reproduce section here: https://github.com/AnkiHubSoftware/ankihub/pull/2313


## Video

https://github.com/user-attachments/assets/639bae88-64bd-40fb-a33d-3890ea921380

pedroven reviewed (approved):
Overall, everything is working, but the result of  the filter of states is not immediately updated after clicking on the button "Add flashcards to reviews"

RisingOrange commented:
@pedroven Good point! I added a commit: [Trigger suspension filter refresh after notes get unsuspended](https://github.com/AnkiHubSoftware/ankihub_addon/pull/1010/commits/6ae19c849999756f915437f84d7bd19f368195d4)
Also added a video of the behavior to the PR description.
